server/controller.js

```diff
@@ -241,13 +242,11 @@ const controllerFactory = ({
 
     const queryString = url.parse(req.url).query
 
-    const { bookingId } = await elite2Api.getDetailsLight(res.locals, offenderNo)
-
-    const data = await elite2Api.get(res.locals, `api/bookings/${bookingId}/caseNotes?${queryString}`)
+    const data = await caseNotesApi.getCaseNotes(res.locals, offenderNo, qs.parse(queryString))
```

Comment:
disappointing that no existing test for this controller, would be good to add one